lib/translation_io/tasks.rb

```diff
@@ -12,35 +13,70 @@
 
   desc "Initialize Translation.io with existing keys/strings."
   task :init => :environment do
-    if client_ready?
+    config = TranslationIO.config
+    if config.multi_domain
+      config.domain_names.each do |domain|
+        config.change_domain(domain)
+        TranslationIO.client = TranslationIO::Client.new(config.api_key, config.endpoint)
+        TranslationIO.client.init
+      end
+    elsif client_ready?
       TranslationIO.client.init
     end
   end
```

Comment:
Same approach used for all rake-tasks.
if config.multi_domain specified, we iterate over each domain, set that to the current domain, and re-run the task.
Otherwise we just run the task